src/components/Chip/Chip.tsx

```diff
@@ -34,48 +36,47 @@ class Chip extends React.PureComponent<Props, any> {
       theme,
     } = this.props;
 
-    let className;
+    const className = classNames(
+      theme.Chip,
+      transparent && theme.transparent
+    );
 
-    if (this.props.markedForDelete) {
-      className = classNames(
-        theme.Chip,
-        theme.markedForDelete,
-        transparent && theme.transparent
-      );
-    } else {
-      className = classNames(
-          theme.Chip,
-          transparent && theme.transparent
-      );
-    }
+    const deleteInstruct = this.props.removable ? 'Press delete to remove this chip' : '';
 
     const chipContents = [(
       image
       ?
-      <img className={theme.Image} src={image.url} alt={image.alt} key="1" />
+      <img className={theme.Image} src={image.url} aria-hidden key="1" />
       : ''
     ),
       <span key="2">
         {this.props.children}
+      <span className={theme.hidden}>{deleteInstruct}</span>
       </span>,
     ];
-
     return (
-      <div className={className}>
+      <div
+        className={className}
+        tabIndex={this.props.tabIndex === 0 ? this.props.tabIndex : -1}
+        onFocus={this.props.autoSuggestMethods ? this.props.autoSuggestMethods.onfocus : ''}
+        onBlur={this.props.autoSuggestMethods ? this.props.autoSuggestMethods.onFocusOut : ''}
```

Comment:
It would be cleaner if we create separate functions for onFucus and onBlur and check and class autoSuggestMethods there, instead of needing this logic here.